tests/test-ptrace.c

```diff
@@ -233,7 +233,11 @@ main (int argc, char **argv)
         fprintf(stderr, "Need to specify a command line for the child\n");
         exit (-1);
       }
+#ifdef __FreeBSD__
```

Comment:
I feel like you need to revert the part above (conditionally) to get the test pass again.

Reply:
Good note, thanks! I'll give that a try.